frontend/src/pages/samples/SamplesPage.tsx

```diff
@@ -25,6 +32,40 @@ export default function SamplesPage() {
             }),
           } as SampleWhere;
         }}
+        customToolbarUI={
+          <>
+            <Button
+              onClick={() => {
+                setColumnDefs(SampleMetadataDetailsColumns);
+              }}
+              size="sm"
+              variant="outline-secondary"
+              active={_.isEqual(columnDefs, SampleMetadataDetailsColumns)}
+            >
+              View SampleMetadata
+            </Button>{" "}
+            <Button
+              onClick={() => {
+                setColumnDefs(ReadOnlyCohortSampleDetailsColumns);
+              }}
+              size="sm"
+              variant="outline-secondary"
+              active={_.isEqual(columnDefs, ReadOnlyCohortSampleDetailsColumns)}
+            >
+              View Tempo
+            </Button>{" "}
+            <Button
+              onClick={() => {
+                setColumnDefs(combinedSampleDetailsColumns);
+              }}
+              size="sm"
+              variant="outline-secondary"
+              active={_.isEqual(columnDefs, combinedSampleDetailsColumns)}
+            >
+              View all
+            </Button>
+          </>
+        }
```

Comment:
When clicked, each button will update the Samples table's column definitions to only include fields of SampleMetadata, Tempo, or all.

The `active` prop adds a background color to the tab that is currently selected to let users know which tab/view they're currently in.